lib/logux/class_finder.rb

```diff
@@ -11,19 +11,21 @@ def initialize(params)
     def find_action_class
       "#{class_namespace}::#{class_name}".constantize
     rescue NameError
-      raise Logux::NoActionError, %(
```

Comment:
I think it's a bad idea to remove the errors raising because we have error handling here:
https://github.com/wilddima/logux_rails/blob/master/app/controllers/logux_controller.rb#L11
So maybe we should handle errors in a different way. For example:
```
case e
when Logux::NoActionError
  # ....
when Logux::NoPolicyError
  # ....
else
  # ....
end
```

However, for us it's errors, and our behaviour shouldn't change even if our response should. We should continue to log this errors and use a callback on them: https://github.com/wilddima/logux_rails/blob/master/app/controllers/logux_controller.rb#L13-L14